components/camera/camera.go

```diff
@@ -140,9 +140,9 @@ type VideoSource interface {
 	// that may have a MIME type hint dictated in the context via gostream.WithMIMETypeHint.
 	Stream(ctx context.Context, errHandlers ...gostream.ErrorHandler) (gostream.VideoStream, error)
 
-	// NextPointCloud returns the next immediately available point cloud, not necessarily one
+	// PointCloud returns the next immediately available point cloud, not necessarily one
 	// a part of a sequence. In the future, there could be streaming of point clouds.
-	NextPointCloud(ctx context.Context) (pointcloud.PointCloud, error)
+	PointCloud(ctx context.Context, extra map[string]interface{}) (pointcloud.PointCloud, error)
```

Comment:
Bro... I just realized this thing has a mimetype and returns bytes too....
https://github.com/viamrobotics/api/blob/a97363bad4477f2a2320c28be0d12447d9a81f89/proto/viam/component/camera/v1/camera.proto#L107

Reply:
no.

Reply:
noooooooooo

Reply:
We are ok to handle `mimetype` inclusion in another PR? There are no consequences wrt data capture by adding in `extra` to the interface method here?

Reply:
> We are ok to handle mimetype inclusion in another PR?

If we are going to break the same interface I'd rather not do it twice.